lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml

```diff
@@ -2460,6 +2460,159 @@ definitions:
+    type: object
+    properties:
+      a1_threshold_rsrp:
+        type: integer
+        format: uint32
+        minimum: 0
+        maximum: 97
+        example: 50
+        default: 50
+      lte_a1_threshold_rsrq:
+        type: integer
+        format: int32
+        example: 2
+      hysteresis:
+        type: integer
+        format: int32
+        minimum: 0
+        maximum: 30
+        example: 3
+      time_to_trigger:
+        type: string
+        example: "ms480"
+      a2_threshold_rsrp:
+        type: integer
+        format: int32
+        minimum: 0
+        maximum: 97
+        example: 5
+      lte_a2_threshold_rsrq:
+        type: integer
+        format: int32
+        example: 6
+      lte_a2_threshold_rsrp_irat_volte:
+        type: integer
+        format: int32
+        example: 7
+      lte_a2_threshold_rsrq_irat_volte:
+        type: integer
+        format: int32
+        example: 8
+      a3_offset:
+        type: integer
+        format: int32
+        minimum: -30
+        maximum: 30
+        example: 10
+      a3_offset_anr:
+        type: integer
+        format: int32
+        example: 10
+      a4_threshold_rsrp:
+        type: integer
+        format: int32
+        example: 50
+      lte_intra_a5_threshold_1_rsrp:
+        type: integer
+        format: int32
+        minimum: 0
+        maximum: 97
+        example: 50
+      lte_intra_a5_threshold_2_rsrp:
+        type: integer
+        format: int32
+        minimum: 0
+        maximum: 97
+        example: 51
+      lte_inter_anr_a5_threshold_1_rsrp:
+        type: integer
+        format: int32
+        minimum: 0
+        maximum: 97
+        example: 55
+      lte_inter_anr_a5_threshold_2_rsrp:
+        type: integer
+        format: int32
+        minimum: 0
+        maximum: 97
+        example: 50
+      b2_threshold1_rsrp:
+        type: integer
+        format: uint32
+        minimum: 0
+        maximum: 97
+        example: 8
+      b2_threshold2_rsrp:
+        type: integer
+        format: int32
+        minimum: -5
+        maximum: 91
+        example: 8
+      b2_geran_irat_threshold:
+        type: integer
+        format: uint32
+        minimum: 0
+        maximum: 63
+        example: 8
+      qrxlevmin_sib1:
+        type: integer
+        format: int32
+        minimum: -70
+        maximum: -22
+        example: 8
+        default: -23
+      qrxlevminoffset:
+        type: integer
+        format: uint32
+        minimum: 1
+        maximum: 8
+        example: 8
+        default: 6
+      s_intrasearch:
+        type: integer
+        format: uint32
+        minimum: 0
+        maximum: 31
+        example: 8
+      s_nonintrasearch:
+        type: integer
+        format: uint32
+        minimum: 0
+        maximum: 31
+        example: 8
+      qrxlevmin_sib3:
+        type: integer
+        format: int32
+        minimum: -70
+        maximum: -22
+        example: 8
```

Comment:
Same as above (example value and allowed range don't match)

Reply:
![image](https://user-images.githubusercontent.com/87301024/154631073-f3f6effa-b29e-4cde-bb91-7a240ccfa458.png)
Modified, thank you!